src/main/scheduler/scheduler_tasks.c

```diff
@@ -32,7 +32,7 @@ cfTask_t cfTasks[TASK_COUNT] = {
         .staticPriority = TASK_PRIORITY_HIGH,
     },
 
-    [TASK_GYROPID] = {
+    [TASK_PID] = {
         .taskName = "PID",
         .subTaskName = "GYRO",
```

Comment:
Yes that was the next step. I still think it is good to add a gyro update rate somewhere in the task command as it does impact the performance.